src/background/data/plannerdata.js

```diff
@@ -568,20 +608,76 @@ window.PlannerData = {
             new PlannerItem(8, SUPPLYTYPE.treasure, PLANNER_TEMPLATES.trueAnima.Dark, 3),
             new PlannerItem(8, PLANNER_ITEMS.rainbowPrism, 250),
             new PlannerItem(8, PLANNER_ITEMS.blueSkyCrystal, 30),
-            new PlannerItem(8, SUPPLYTYPE.evolution, 20004, 1) //Gold brick 1
-            //crystal 500
+            new PlannerItem(8, SUPPLYTYPE.evolution, 20004, 1), //Gold brick 1
+            //crystal 600
+
+           //8 Crafting an uncapped Silver relic
+            new PlannerItem(10, SUPPLYTYPE.treasure, 5011, 300),
+            new PlannerItem(10, SUPPLYTYPE.treasure, 5021, 300),
+            new PlannerItem(10, SUPPLYTYPE.treasure, 5031, 300),
+            new PlannerItem(10, SUPPLYTYPE.treasure, 5041, 300),
+            new PlannerItem(10, SUPPLYTYPE.treasure, 5051, 300),
+            new PlannerItem(10, SUPPLYTYPE.treasure, 5061, 300),
+           
+           //9 Creating a Golden Relic
```

Comment:
Oh, the PLANNER_ITEMS and PLANNER_TEMPLATES here should use the 3 argument version of the constructor. The type (SUPPLYTYPE.treasure) should be removed.